### PR TITLE
move latest ocp version to 4.13

### PR DESCRIPTION
**Pull request by acornett21**
- OCP 4.13 is GA and in production, updating our bundle validate logic
- Adding in 4.14 to metrix so we have a 'next' version to lookup

### CODE REVIEW

**Review by komish — APPROVED** [review]
/lgtm

**Review by skattoju — APPROVED** [review]
/lgtm

**Review by skattoju — APPROVED** [review]

File: internal/bundle/bundle.go [hunk 1]
```diff
@@ -27,9 +27,10 @@ var ocpToKubeVersion = map[string]string{
 	"4.11": "1.24",
 	"4.12": "1.25",
 	"4.13": "1.26",
+	"4.14": "1.27",
```
[the remark this answers is not in the document]
Comment (acornett21) [reply internal/bundle/bundle.go:30]: @tkrishtop That seems to work, is this code somewhere else as well?

[the remark this answers is not in the document]
Comment (tkrishtop) [reply internal/bundle/bundle.go:30]: @acornett21 that's my translation of DCI Python code to Go: https://github.com/redhat-cip/dci-openshift-app-agent/blame/master/roles/deprecated-api/filter_plugins/dump_api.py#L31-L32 
The idea with 2.87 is not mine, though. I've seen it somewhere else and liked it.

[the remark this answers is not in the document]
Comment (acornett21) [reply internal/bundle/bundle.go:30]: @bcrochet @komish What are your thoughts on that logic? My take is it is one less thing to remember, but on the other hand the map is nice so we guarantee the `next` version is correct.

**Pull request merged**